docs/updates/2020-08-07-announce-book.md

```diff
@@ -0,0 +1,108 @@
+
+The new Jupyter Book has the following main features (with links to the relevant documentation for each):
+
+* **[Write publication-quality content in markdown](https://jupyterbook.org/content-types/markdown)**. You can write in either Jupyter markdown, or in [MyST Markdown, an extended flavor of markdown for scientific publishing](https://jupyterbook.org/content/myst).  This includes support for rich syntax such as [citations and cross-references](https://jupyterbook.org/content/citations), [math and equations](https://jupyterbook.org/content/math), and [figures](https://jupyterbook.org/content/figures).
+* **[Write content in Jupyter Notebooks](https://jupyterbook.org/content-types/notebooks)**, allowing you to include your code and outputs in your book. You can also write [notebooks entirely in markdown](https://jupyterbook.org/content-types/myst-notebooks) to execute when you build your book.
+* **[Execute and cache your book's content](https://jupyterbook.org/content/execute)**. For `.ipynb` and markdown notebooks, execute code and insert the latest outputs into your book. In addition, [cache and re-use](https://jupyterbook.org/content/execute.html#cacheing-the-notebook-execution) outputs to be used later.
+* **[Insert notebook outputs into your content](https://jupyterbook.org/content/glue)**. Generate outputs as you build your documentation, and insert them in-line with your content across pages.
+* **[Add interactivity to your book](https://jupyterbook.org/interactive/launchbuttons)**. You can [toggle visibility of cells](https://jupyterbook.org/interactive/hiding), [connect with an online service like Binder](https://jupyterbook.org/interactive/launchbuttons), and include [interactive outputs from Jupyter](https://jupyterbook.org/interactive/interactive).
+* **[Generate a variety of outputs](https://jupyterbook.org/start/build)**, including single- and multi-page websites, as well [as PDF outputs](https://jupyterbook.org/advanced/pdf).
+* **[Build your book with a simple CLI](https://jupyterbook.org/reference/cli)** to quickly generate your books with one command, like so: `jupyter-book build mybook/`
+
+These are just a few of the major changes that we've made. For a more complete idea of what you can do, [check out the Jupyter Book documentation](https://jupyterbook.org)
+
+## Major changes
+
+In the next few sections we'll talk about some major changes that you may notice in Jupyter Book.
+
+### An enhanced flavor of markdown
+
+The biggest enhancement to Jupyter Book is support for the [MyST Markdown language](https://myst-parser.readthedocs.io/en/latest/). MyST is a flavor of markdown that implements all of the features of Sphinx, allowing you to write scientific publications in markdown. It draws inspiration from RMarkdown and the reStructuredText ecosystem of tools. Anything you can do in Sphinx, you can do with MyST as well.
```

Comment:
Include the acronym expansion somewhere? i.e. Markedly Structured Text (MyST)

Reply:
link to sphinx docs and/or mention what it is (e.g. the sphinx static site generator)?

Reply:
ah you do mention what sphinx is later on. Maybe better earlier but not sure

Reply:
added a bit more explanation of sphinx + a link, and added acronym expansion